$NetBSD: patch-ad,v 1.9 2009/07/27 12:59:44 obache Exp $

PerlIO mutex must be initialized before trying to fiddle with the
interpreter.

--- src/perl/perl-core.c.orig	2008-12-04 13:45:24.000000000 +0100
+++ src/perl/perl-core.c
@@ -452,6 +452,8 @@ void perl_core_init(void)
 	perl_signals_init();
         signal_add_last("script error", (SIGNAL_FUNC) sig_script_error);
 
+	PERLIO_INIT;
+
 	perl_scripts_init();
 
 	if (irssi_init_finished)
